scripts/install/install.js

```diff
@@ -1,2 +1,8 @@
 var fs = require('fs')
-fs.createReadStream('scripts/install/tsqllint').pipe(fs.createWriteStream(`${process.env.APPDATA}/npm/tsqllint`))
\ No newline at end of file
+const execSync = require('child_process').execSync;
+
+// calls powershell script to install missing dependencies
+execSync(`call ${process.env.APPDATA}/npm/node_modules/tsqllint/scripts/install/install-dependencies.cmd`);
```

Comment:
`execSync` doesn't exist in Node.js 0.10 (only noted since that's what the base image comes installed with). LGTM if you're OK with that. May not be an issue if the default install has a broken PATH anyway, so a new install would do it good.